tests/Microsoft.Azure.CognitiveServices.Samples.Test/Microsoft.Azure.CognitiveServices.Samples.Test.csproj

```diff
@@ -10,6 +10,7 @@
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.7.0" />
     <PackageReference Include="MSTest.TestAdapter" Version="1.2.1" />
     <PackageReference Include="MSTest.TestFramework" Version="1.2.1" />
+    <PackageReference Include="NewtonSoft.Json" Version="10.0.3" />
```

Comment:
What will happen if you don't add this?

Reply:
@yangyuan The tests will fail for text analytics since its looking for Newtonsoft version 10.0.0. Microsoft.Net.Test.Sdk has Newtonsoft version 9.0.0